Assets/Scripts/MapGen/MapGenerator.cs

```diff
@@ -295,7 +296,7 @@ Sprite SaveImageAsset(Texture2D texture, string name)
             System.IO.File.WriteAllBytes(absolutePath, byteArray);
             ApplySpriteDefaults(relativePath);
             AssetDatabase.ImportAsset(relativePath, ImportAssetOptions.DontDownloadFromCacheServer);
-            ResetSpriteImporter(relativePath);
+            // ResetSpriteImporter(relativePath);
```

Comment:
This was causing weirdness with the Sprite meta settings. In the inspector, the pixels per unit would show `100` even though the actual value was `1`.

This might make it so that `.png` imports default to 1 pixels per unit... let me know if this is an issue and I'll try and look into it.

Unity makes it maddeningly difficult to edit Sprite data unfortunately. Tried the SerializeObject route previously and that did not work. Though in hindsight it may be that the asset hadn't been imported yet. Maybe it will work after import...

Some background - Unity saves `.png` (and other image formats) as Texture2Ds, but adds a Sprite wrapper on top of it. The Sprite wrapper is what contains the extra metadata like pixelsPerUnit, filterMode, etc. It would be SO nice to programmatically change these properties without having to manually change them in the inspector each time. Alas...